cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShowProceduresTest.scala

```diff
@@ -28,60 +28,158 @@ class ShowProceduresTest extends DocumentingTest {
+
+[NOTE]
+====
+When using the `RETURN` clause, the `YIELD` clause is mandatory and may not be omitted.
+====
+""".stripMargin('#'))
+    }
+    section("Listing all procedures") {
+      p(
+        """To list all available procedures with the default output columns, the `SHOW PROCEDURES` command can be used.
+          #If all columns are required, use `SHOW PROCEDURES YIELD *`.""".stripMargin('#'))
+      query("SHOW PROCEDURES", ResultAssertions(p => {
+        p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem")
+      })) {
+        limitedResultTable(List("name", "description", "mode", "worksOnSystem"), 15)
       }
-      section("Listing procedures with filtering on output columns") {
-        p(
-          """The listed procedures can be filtered in multiple ways, one way is to use the `WHERE` clause.
-            #For example getting the name of all admin procedures:""".stripMargin('#'))
-        query("SHOW PROCEDURES YIELD name, admin WHERE admin", ResultAssertions(p => {
-          p.columns should contain theSameElementsAs Array("name", "admin")
-          p.columnAs[Boolean]("admin").foreach(_ should be(true))
-        })) {
-          limitedResultTable(List("name", "admin"), 7)
-        }
+    }
+    section("Listing procedures with filtering on output columns") {
+      p(
+        """The listed procedures can be filtered in multiple ways, one way is to use the `WHERE` clause.
+          #For example getting the name of all admin procedures:""".stripMargin('#'))
+      query("SHOW PROCEDURES YIELD name, admin WHERE admin", ResultAssertions(p => {
+        p.columns should contain theSameElementsAs Array("name", "admin")
+        p.columnAs[Boolean]("admin").foreach(_ should be(true))
+      })) {
+        limitedResultTable(List("name", "admin"), 7)
+      }
+    }
+    section("Listing procedures with other filtering") {
+      database("system")
+      initQueries(
+        "CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED"
+      )
+      database("neo4j")
+      p(
+        """The listed procedures can also be filtered by whether a user can execute them.
+          #This filtering is only available through the `EXECUTABLE` clause and not through the `WHERE` clause.
+          #This is due to using the users privileges instead of just filtering on the available output columns.""".stripMargin('#'))
+      p(
+        """There are two versions of the `EXECUTABLE` clause.
+          #The first one is to filter for the current user:""".stripMargin('#'))
+      login("jake", "abc123")
+      query("SHOW PROCEDURES EXECUTABLE BY CURRENT USER YIELD *", ResultAssertions(p => {
+        p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem", "signature", "argumentDescription", "returnDescription", "admin", "rolesExecution", "rolesBoostedExecution", "option")
+        p.columnAs[List[String]]("rolesExecution").foreach(_ should be(null))
```

Comment:
All descriptions render as code blocks and within quotation marks. It's not strictly wrong, but it doesn't look so nice.